src/services/services.ts

```diff
@@ -3653,6 +3710,10 @@ namespace ts {
 
             let completionData = getCompletionData(fileName, position);
             if (!completionData) {
+                let entries = getJsDocCompletionEntries(fileName, position);

```

Comment:
i do not understand why this is done here. earlier in getCompletionData you already detected you are in a jsdoc comment, why not just return this there?